package/packageConfig.js

```diff
@@ -70,6 +71,23 @@ async function addDependency(dep) {
   writePackageConfig(config);
 }
 
+/**
+ * Update the dependency if not listed in package.json
+ * 
+ * @param {string} pkg - id of the dependency package
+ * @returns {void}
+ */
+
+async function addDependencyIfMissing(pkg) {
+  if (!getPackageDependencies().includes(pkg)) {
```

Comment:
Instead of having a function that returns an array, and then performing an `includes` on it, why not create a function that verifies if the dependency exists? It would become much more readable.

Reply:
@andreyluiz you mean to check if the dependency exists on npm?